android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -147,20 +139,36 @@ private long getBinaryResourcesModifiedTime() {
     }
 
     public static String getBundleUrl() {
```

Comment:
Should we mark these as being @Deprecated? That way the code and usage is clearer that we expected you to just the newer methods?